internal/configurations/rest_controller_configuration.go

```diff
@@ -9,8 +9,12 @@ import (
 // RESTControllerConfiguration defines a struct with required environment variables for rest controller.
 type RESTControllerConfiguration struct {
 	OIDCCallbackURL    string        `envconfig:"OIDC_CALLBACK_URL" default:"/auth/oidc/callback"`
-	AuthServiceURL     string        `envconfig:"AUTHSERVICE_URL" default:"/auth"`
+	AuthServiceURL     string        `envconfig:"AUTHSERVICE_URL" default:"/"`
+	LogoutURL          string        `envconfig:"LOGOUT_URL" default:"/logout"`
 	HTTPRequestTimeout time.Duration `envconfig:"HTTP_REQUEST_TIMEOUT" default:"30s"`
+	AuthHeader         string        `envconfig:"AUTH_HEADER" default:"Authorization"`
```

Comment:
first, i don't think this belongs to the controller, this is handler responsability, right?
second, i don't think this should be configured, if so, we must create an handler responsible for that.

Reply:
hmmm, I agree. thanks Portela.